store/details_v2.go

```diff
@@ -89,9 +90,9 @@ func infoFromStoreSnap(d *storeSnap) (*snap.Info, error) {
 	info.RealName = d.Name
 	info.Revision = snap.R(d.Revision)
 	info.SnapID = d.SnapID
-	info.EditedTitle = d.Title
-	info.EditedSummary = d.Summary
-	info.EditedDescription = d.Description
+	info.EditedTitle = d.Title.Clean()
```

Comment:
Could we use String() instead of clean to make this more hassle-free?

Reply:
I'm not sure how it would make it more hassle-free; it'd make it more susceptible to using it as a Stringer and passed to, say, `fmt.Sprint()`, but as `Clean()` could in theory be expensive, I thought it best to explicitly avoid that.

Promising that the cleaning will always be done upfront would be another approach, but I'm not sure it's better.

Reply:
We could make String() call Clean() and cache it, this is a read-only thing after all.

Reply:
We could. What's the advantage of that?